src/card-stack/card-stack.html

```diff
@@ -49,16 +49,32 @@
         float: left;
         position: absolute;
         margin: 0px;
+        z-index:0;
+      }
+      .foundation-stack game-card {
+        margin-top:-176px;
+        z-index:1;
       }
 
-
+      .bottom{
+        bottom:0px;
+      }
+      .left{
+        left:0px;
+      }
+      .right{
+        right:0px;
+      }
+      .top{
+        top:0px;
+      }
       /*.stack game-card[position="1"] { top: 40px; }
       .stack game-card[position="2"] { top: 80px; }
       .stack game-card[position="3"] { top: 120px; }*/
 
     </style>
 
-    <div class="stack">
+    <div class$="[[_stackClass()]]">
```

Comment:
You need to pass in the dependent variables here to ensure the binding is updated when those variables change. Looking at the `_stackClass()` function it seems dependent on `foundation`. This would not update when that variable changes currently.